cmd/sealer/cmd/cluster/scale-up.go

```diff
@@ -75,13 +76,27 @@ func NewScaleUpCmd() *cobra.Command {
 			if err != nil {
 				return err
 			}
+			defaultCf, err = clusterfile.NewClusterFile(clusterFileData)
+			if err != nil {
+				return err
+			}
 
 			cluster := cf.GetCluster()
 			if err = utils.ConstructClusterForScaleUp(&cluster, ScaleUpFlags, scaleUpMasterIPList, scaleUpNodeIPList); err != nil {
 				return err
 			}
 			cf.SetCluster(cluster)
 
+			defer func() {
+				if err == nil {
+					return
+				}
+				//if there exits an error,restore the ClusterFile to the default file
```

Comment:
Let's name the function called "rollback or something"? Because I see you call this in two places.